products/tailwind-css.md

```diff
@@ -0,0 +1,142 @@
+eolColumn: Support
+
+auto:
+  methods:
+  -   git: https://github.com/tailwindlabs/tailwindcss
+
+releases:
+-   releaseCycle: "4.1"
+    releaseDate: 2025-04-01
+    eol: false
+    latest: '4.1.11'
+    latestReleaseDate: 2025-06-26
+
+-   releaseCycle: "4.0"
+    releaseDate: 2025-01-21
+    eol: false
+    latest: '4.0.17'
+    latestReleaseDate: 2025-03-26
+
+-   releaseCycle: "3.4"
+    releaseDate: 2023-12-18
+    eol: false
+    latest: '3.4.17'
+    latestReleaseDate: 2024-12-17
+
+-   releaseCycle: "3.3"
+    releaseDate: 2023-03-28
+    eol: 2024-12-19
+    latest: '3.3.7'
+    latestReleaseDate: 2023-12-18
+
+-   releaseCycle: "3.2"
+    releaseDate: 2022-10-19
+    eol: 2024-03-28
+    latest: '3.2.7'
+    latestReleaseDate: 2023-02-16
+
+-   releaseCycle: "3.1"
+    releaseDate: 2022-06-08
+    eol: 2023-10-19
+    latest: '3.1.8'
+    latestReleaseDate: 2022-08-05
+
+-   releaseCycle: "3.0"
+    releaseDate: 2021-12-09
+    eol: 2023-06-07
+    latest: '3.0.24'
+    latestReleaseDate: 2022-04-12
+
+-   releaseCycle: "2.2"
+    releaseDate: 2021-06-17
+    eol: 2022-12-09
+    latest: '2.2.19'
+    latestReleaseDate: 2021-10-29
+
+-   releaseCycle: "2.1"
+    releaseDate: 2021-04-05
+    eol: 2022-05-20
+    latest: '2.1.4'
+    latestReleaseDate: 2021-06-02
+
+-   releaseCycle: "2.0"
+    releaseDate: 2020-11-18
+    eol: 2022-04-05
+    latest: '2.0.5'
+    latestReleaseDate: 2021-04-05
+
+-   releaseCycle: "1.9"
+    releaseDate: 2020-10-12
+    eol: 2021-11-18
+    latest: '1.9.6'
+    latestReleaseDate: 2020-10-23
+
+-   releaseCycle: "1.8"
+    releaseDate: 2020-09-04
+    eol: 2021-10-13
+    latest: '1.8.13'
+    latestReleaseDate: 2020-10-09
+
+-   releaseCycle: "1.7"
+    releaseDate: 2020-08-18
+    eol: 2021-08-17
+    latest: '1.7.6'
+    latestReleaseDate: 2020-08-29
+
+-   releaseCycle: "1.6"
+    releaseDate: 2020-07-28
+    eol: 2021-07-15
+    latest: '1.6.3'
+    latestReleaseDate: 2020-08-18
+
+-   releaseCycle: "1.5"
+    releaseDate: 2020-07-15
+    eol: 2021-05-28
+    latest: '1.5.2'
+    latestReleaseDate: 2020-07-21
+
+-   releaseCycle: "1.4"
+    releaseDate: 2020-04-29
+    eol: 2021-07-21
+    latest: '1.4.6'
+    latestReleaseDate: 2020-05-08
+
+-   releaseCycle: "1.3"
+    releaseDate: 2020-04-20
+    eol: 2021-03-18
+    latest: '1.3.5'
+    latestReleaseDate: 2020-04-23
+
+-   releaseCycle: "1.2"
+    releaseDate: 2020-02-05
+    eol: 2021-02-05
+    latest: '1.2.0'
+    latestReleaseDate: 2020-02-05
+
+-   releaseCycle: "1.1"
+    releaseDate: 2019-08-06
+    eol: 2021-02-05
+    latest: '1.1.4'
+    latestReleaseDate: 2019-11-25
+
+-   releaseCycle: "1.0"
+    releaseDate: 2019-05-13
+    eol: 2020-08-06
+    latest: '1.0.6'
+    latestReleaseDate: 2019-07-31
+
+---
+
+> [Tailwind CSS](https://tailwindcss.com/) is a utility-first CSS framework for rapidly building modern websites without ever leaving your HTML.
+
+Only v4 and latest v3 are considered as not deprecated versions.
```

Comment:
What is the source of this information ?

Based on the latest version of each minor release, it seems to me that only the latest minor is supported.

Reply:
I changed the description to indicate that it's not documented and only the latest minor is supported. I also updated the eol dates accordingly.